tests/test_transform.py

```diff
@@ -31,3 +34,29 @@ def test_transform_xml_rules_to_yaml():
         )
     except ValidationError as e:
         assert True is False, 'The validation of the generated YAML rule failed. ' + e.message
+
+
+# Polarion TC 376
+def test_transform_code_with_openrewrite():
+    kantra_path = os.getenv(constants.KANTRA_CLI_PATH)
+    complete_duke_app_path = os.path.join(os.getenv(constants.PROJECT_PATH), 'data/applications', 'complete-duke.zip')
```

Comment:
Instead of "complete_duke_app_path" we can give
just "application_path"

Reply:
Solved in a1a2da7352701ec3a24a8116ca2afff571cc934b